Tasks/IISWebAppManagementOnMachineGroup/task.json

```diff
@@ -1,650 +1,684 @@
+            "options": {
+                "v4.0": "v4.0",
+                "v2.0": "v2.0",
+                "No Managed Code": "No Managed Code"
+            }
+        },
+        {
+            "name": "PipeLineModeForWebsite",
+            "type": "pickList",
+            "label": "Managed pipeline mode",
+            "defaultValue": "Integrated",
+            "required": true,
+            "groupName": "ApplicationPoolForWebsite",
+            "helpMarkDown": "Select the managed pipeline mode that specifies how IIS processes requests for managed content. Use classic mode only when the applications in the application pool cannot run in the Integrated mode.",
+            "options": {
+                "Integrated": "Integrated",
+                "Classic": "Classic"
+            }
+        },
+        {
+            "name": "AppPoolIdentityForWebsite",
+            "type": "pickList",
+            "label": "Identity",
+            "defaultValue": "ApplicationPoolIdentity",
+            "required": true,
+            "groupName": "ApplicationPoolForWebsite",
+            "helpMarkDown": "Configure the account under which an application pool's worker process runs. Select one of the predefined security accounts or configure a custom account.",
+            "options": {
+                "ApplicationPoolIdentity": "Application Pool Identity",
+                "LocalService": "Local Service",
+                "LocalSystem": "Local System",
+                "NetworkService": "Network Service",
+                "SpecificUser": "Custom Account"
+            }
+        },
+        {
+            "name": "AppPoolUsernameForWebsite",
+            "type": "string",
+            "label": "Username",
+            "defaultValue": "",
+            "required": true,
+            "groupName": "ApplicationPoolForWebsite",
+            "visibleRule": "AppPoolIdentityForWebsite = SpecificUser",
+            "helpMarkDown": "Provide the username of the custom account that you want to use."
+        },
+        {
+            "name": "AppPoolPasswordForWebsite",
+            "type": "string",
+            "label": "Password",
+            "defaultValue": "",
+            "required": false,
+            "groupName": "ApplicationPoolForWebsite",
+            "helpMarkDown": "Provide the password for custom account. <br/>The best practice is to create a variable in the Build or Release definition, and mark it as 'Secret' to secure it, and then use it here, like '$(userCredentials)'. <br> Note: Special characters in password are interpreted as per <a href=\"https://go.microsoft.com/fwlink/?linkid=843470\">command-line arguments</a>",
+            "visibleRule": "AppPoolIdentityForWebsite = SpecificUser"
+        },
+        {
+            "name": "BasicAuthenticationForWebsite",
```

Comment:
No it is check box you can have more than once auth enabled. I sent spec screen shot to you